geodata/agglomeration_2012.json

```diff
@@ -1,158 +1,6681 @@
 {
-"type": "FeatureCollection",
```

Comment:
I'd recomend excluding all files in geodata/ folder from being formatted by prettier. These are data files (that are either downloaded from the internet or processed by scripts locally), not to be consumed or edited by a human.